CODEOWNERS

```diff
@@ -9,6 +9,7 @@
 /core/internal @samsondav @jmank88
 
 # Chains
+/common @smartcontractkit/integrations
```

Comment:
@prashantkumar1982 does it make sense to tag the whole team like this? Or would we rather list individuals for now?

Reply:
Merging for now but feel free to refine this as you see fit.